dts/arm/st/f3/stm32f303.dtsi

```diff
@@ -32,6 +32,23 @@
 			status = "disabled";
 		};
 
+		i2c3: i2c3@4007800 {
```

Comment:
Please move node to `dts/arm/st/f3/stm32f303Xe.dtsi` as it is not available in other stm32f303 variants.